k-distribution/include/kore/prelude.kore

```diff
@@ -25,7 +25,7 @@ module KSEQ
 endmodule []
 
 module INJ
-  symbol inj{From,To}(From) : To []
+  symbol inj{From,To}(From) : To [injection{}()]
```

Comment:
I would name this `sortInjection` to avoid it being confused with `injective`.  What do you think?